ansible/group_vars/all

```diff
@@ -51,7 +59,10 @@ nginx_vhosts:
       }
 
       location /admin {
+        index index.html;
```

Comment:
We haven't talked about workflow, but going to http://biblebox.local/admin results in the admin page with a non-functional menu (and no front matter). The URL is shown in the browser as http://biblebox.local/admin#   . http://biblebox.local/admin/ is fine, however, so we're probably just missing a 302 somewhere.